src/main/java/org/apache/aurora/scheduler/preemptor/PreemptionVictimFilter.java

```diff
@@ -169,7 +171,9 @@ public int compare(ResourceBag left, ResourceBag right) {
       // This enforces the precondition that all of the resources are from the same host. We need to
       // get the host for the schedulingFilter.
       Set<String> hosts = ImmutableSet.<String>builder()
-          .addAll(Iterables.transform(possibleVictims, VICTIM_TO_HOST))
+          .addAll(StreamSupport
```

Comment:
Can't see if this violates the line length but if it doesn't, just to keep consistency with the rest of the codebase, it'd be better to keep the first chained call on the original line followed by the rest each on one line.
```suggestion
          .addAll(StreamSupport.stream(possibleVictims.spliterator(), false)
              .map(VICTIM_TO_HOST).collect(Collectors.toSet()))
          
```